swagger.yaml

```diff
@@ -536,17 +536,36 @@ definitions:
   FilterOutputResponse:
     description: "A model for the response body when retrieving a filter output"
     allOf:
-    - type: object
-      properties:
-        downloads:
-          $ref: '#/definitions/Downloads'
-        state:
-          description: "This describes the status of the filter."
-          enum:
-            - "completed"
-            - "created"
-          type: string
-    - $ref: '#/definitions/JobState'
+      - type: object
+        properties:
+          downloads:
+            $ref: '#/definitions/Downloads'
+          state:
+            description: "This describes the status of the filter."
+            enum:
+              - "completed"
+              - "created"
+            type: string
+      - type: object
+        properties:
+          links:
+            allOf:
+              - $ref: '#/definitions/FilterLinks'
+              - type: object
+                properties:
+                  filter_blueprint:
+                    allOf:
```

Comment:
No need to include `allOf` here as there is only one object to inherit for a `filter_blueprint`